yt/utilities/exceptions.py

```diff
@@ -864,6 +867,19 @@ def __str__(self):
         return msg
 
 
+class YTConfigError(YTException):
+    def __init__(self, key: str, choices: Optional[Any] = None) -> None:
```

Comment:
Isn't `Optional[Any]` equivalent to `Any`?

Reply:
Oh my. Of course it is ! Well played

Reply:
I think I got a more useful annotation now.